api/routes/project.js

```diff
@@ -6,38 +6,50 @@ var isAuthenticated = require('../utils/authentication');
 var mongoosePaginate = require('mongoose-paginate');
 
 // retrieves all projects
-router.get('/', function(req, res) {
-  Project.paginate({}, req.body.options, function(err, result) {
-    if (err) { return res.send('Error retrieving project: ' + err) };
-    res.json(result);
+router.get('/', function (req, res) {
+  Project.paginate({}, req.body.options, function (err, result) {
+    if (err) {
+      return res.send('Error retrieving project: ' + err)
+    } else {
+      res.setHeader("Content-Type", "application/json");
+      res.json({ projects: result, message: 'Succesfully retrieved projects' });
+    }
   })
 });
 
 // retrieves all projects
-router.post('/filter', function(req, res) {
-  Project.paginate({}, req.body.options, function(err, result) {
-    if (err) { return res.send('Error retrieving project: ' + err) };
-    res.json(result);
+router.post('/filter', function (req, res) {
+  Project.paginate({}, req.body.options, function (err, result) {
```

Comment:
Filtering functionality not implemented, maybe put a comment in signifying it is a placeholder.